### PR TITLE
TB - HIV  - Birth - Death measures | ANC monthly measure

### PR DESCRIPTION
**Pull request by maimoonak**
Fixes ANC, TB, HIV Measures with monthly reporting

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

**Review by f-odhiambo — APPROVED** [review]

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt [hunk 1]
```diff
@@ -250,16 +253,20 @@ constructor(
...
-        )
-      }
+          kotlin.runCatching {
+            fhirOperator.evaluateMeasure(
+              measureUrl = measureUrl,
+              start = startDateFormatted,
+              end = endDateFormatted,
+              reportType = POPULATION,
+              subject = null,
+              practitioner = null /*practitionerDetails?.id*/,
```
Comment (pld) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt:263]: what's w/comment here? we should use // if we still a comment here and add some text to explain

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt [hunk 2]
```diff
@@ -276,35 +283,82 @@ constructor(
...
-              val denominator = reportGroup.findPopulation(NUMERATOR)?.count ?: 0
-              val percentage =
-                if (denominator == 0) null else numerator.toDouble().div(denominator) * 100
-              val count = "$numerator/$denominator"
-              MeasureReportIndividualResult(
-                title = text,
-                percentage = percentage?.roundToInt()?.toString() ?: "0",
-                count = count
-              )
+        // Measure Report :
```
Comment (pld) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt:286]: fine to remove

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt (hunk 1 above)
Comment (maimoonak) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt]: Added TODO and ticket

Comment (maimoonak) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt]: Looks like it is fixed in latest main. Fixed

**Review by pld — APPROVED** [review]
lgtm! let's bump the test coverage then merge

**Review by pld — APPROVED** [review]

File: android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/MeasureExtensions.kt [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+ */
+
+package org.smartregister.fhircore.engine.util.extension
+
+import org.apache.commons.lang3.StringUtils
+import org.hl7.fhir.r4.model.MeasureReport
+import org.opencds.cqf.cql.evaluator.measure.common.MeasurePopulationType
+
+val MONTH_NAMES =
+  listOf("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec")
```
Comment (ellykits) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/MeasureExtensions.kt:24]: Can we have a followup PRs for migrating these hard coded month strings to `string.xml` for purposes of translation.

**Review by ellykits — APPROVED** [review]

File: android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/MeasureExtensions.kt (hunk 3 above)
Comment (maimoonak) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/MeasureExtensions.kt:24]: https://github.com/opensrp/fhircore/issues/1689

**Pull request merged**